### PR TITLE
[DUOS][risk=no] Added Translate Page

### PR DESCRIPTION
**Pull request by lu-c**
It's not linked to anywhere within the duos site, you have to type it into the url bar

----
Have you read [Terra's Contributing Guide](https://github.com/DataBiosphere/terra-ui/wiki/Contributor-Guide) lately? If not, do that first.

- Label PR with a Jira ticket number and include a link to the ticket
- Label PR with a security risk modifier [no, low, medium, high]
- PR describes scope of changes
- Get a minimum of one thumbs worth of review, preferably two if enough team members are available
- Get PO sign-off for all non-trivial UI or workflow changes
- Verify all tests go green
- Test this change deployed correctly and works on dev environment after deployment

### CODE REVIEW

File: src/Routes.js [hunk 1]
```diff
@@ -95,6 +96,7 @@ const Routes = (props) => (
       rolesAllowed={[USER_ROLES.all]} />
     <AuthenticatedRoute path="/tos_acceptance" component={TermsOfServiceAcceptance} props={props} rolesAllowed={[USER_ROLES.all]} />
     {(props.env === 'local' || props.env === 'dev') && <AuthenticatedRoute path="/data_submission_form" component={DataSubmissionForm} props={props} rolesAllowed={[USER_ROLES.all]} />}
+    <AuthenticatedRoute path="/translate" component={Translator} props={props} rolesAllowed={[USER_ROLES.researcher]}/>
```
Comment (JVThomas) [inline src/Routes.js]: Since this is an experimental feature, enable it when `env === local`

Comment (rushtong) [reply src/Routes.js]: I think Gabe is very hopeful to show this off in conjunction with https://github.com/DataBiosphere/consent-ontology/pull/671 once that is approved

Comment (JVThomas) [reply src/Routes.js]: So then should we just limit it to just `dev` and `staging` for now to make it demoable?

Comment (rushtong) [reply src/Routes.js]: Seems reasonable to me!

**Review by JVThomas — APPROVED** [review]
Outside of the missing env restrictions mentioned above the code looks good. We may need to make adjustments once the ontology endpoint is implemented, but I doubt it'll be anything that'll drastically change this code.

File: src/Routes.js (hunk 1 above)
Comment (lu-c) [reply src/Routes.js]: ok! updated!

**Pull request merged**